analytics.js

```diff
@@ -297,6 +305,15 @@ var Analytics = {
                     result.totals.ie_version[version] += parseInt(result.data[i].visits);
                 }
             }
+            if (_.startsWith(report.name, "top-countries")) {
+                // Extract the total visits from the `United States`
+                for (var i=0; i < result.data.length; i++) {
+                    if (result.data[i].country === "United States") {
+                        result.totals.us_visits = result.data[i].visits;
+                    }
```

Comment:
To be clear, I see the value here, in not having to do a search through the whole list. I think it'd be best to avoid special-case code unless it's a very clear help to client-side code.